blocks/blocks.php

```diff
@@ -13,7 +13,7 @@
  * @return string Version number.
  */
 function get_block_version() {
-	return '0.2.0';
```

Comment:
Is this being bumped intentionally? What changed?

Reply:
I meant to get it in line with the plugin's version really, that's why I bumped it

Reply:
It's meant as a signal for breaking changes in the blocks schema. The comment could probably be more explicit.

Reply:
Hmm, I get that. This would probably be better to bump when we add a GraphQL layer on top of the block data api.

I'll revert this version bump and note the purpose behind this version number as well.

Reply:
Reverted it, and updated the Changelog entry with that as well. Thanks for pointing it out, and for explaining it. I've updated the comment as well